docs/asset_pbr_gap_analysis.md

```
@@ -0,0 +1,62 @@
+# Asset & Texture PBR Pipeline Gap Analysis
+
+## Current State Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)